### PR TITLE
Added Vsm Client

### PR DESCRIPTION
**Pull request by abhijeetmanhas**
Split of PR #3763 .

### CODE REVIEW

File: sunpy/net/attrs.py [hunk 1]
```diff
@@ -28,13 +28,15 @@
...
 # Trick the docs into thinking these attrs are defined in here.
-for _a in (Time, Instrument, Wavelength, Level, Sample, Detector, Resolution):
+for _a in (Time, Instrument, Wavelength, Level, Sample, Detector, Resolution,
+           Physobs):
     _a.__module__ = __name__
 
 
 __all__ = ['Time', 'Instrument', 'Wavelength', 'Level', 'Sample', 'Detector', 'Resolution',
-           'vso', 'jsoc', 'goes']
+           'vso', 'jsoc', 'goes', 'Physobs']
```
Comment (dstansby) [inline sunpy/net/attrs.py]: Why have has `Physobs` been added here? I think this might be why the doc build currently isn't working.

Comment (abhijeetmanhas) [reply sunpy/net/attrs.py]: I think this might have been solved in Fido refactoring PR by @cadair, I have to check,

**Comment by dstansby** [comment]
Could you rebase this on the current master branch? I think that will fix some of the tests.

**Comment by abhijeetmanhas** [comment]
@dstansby rebased, still some ci all failing

**Comment by abhijeetmanhas** [comment]
@dstansby all checks have passed, let me know for any other changes to this PR

**Pull request closed without merging**